docs/getting-started.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+A Couchbase-Elasticsearch data-replication system consists of three principal components:
+
+- a **Couchbase Server-cluster** of one or more nodes
+- an **Elasticsearch cluster** of one or more nodes
+- the **Elasticsearch Transport Plug-in**, installed in the Elasticsearch environment.
+
+This section provides a step-by-step procedure, whereby the three components can be installed, configured, and run.
+
+## Pre-requisites
+
+- Ubuntu 14.04 or 16.04
+- Java
+
+## Installation
+
+1. Download the Elasticsearch 2.4.0 package for Ubuntu.
+
+	```bash
+	$ wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-2.4.6.deb
+	```
+
+2. Install the package.
+
+	```bash
+	$ dpkg -i elasticsearch-2.4.6.deb
+	```
+
+3. Navigate to the Elasticsearch installation directory.
+
+	```bash
+	$ cd /usr/share/elasticsearch
+	```
+
+4. Download and install the plug-in package.
+
+	```bash
+	$ bin/plugin install https://github.com/couchbaselabs/elasticsearch-transport-couchbase/releases/download/3.0.0-cypress/elasticsearch-transport-couchbase-3.0.0-alder-es2.4.6.zip
+	```
+
+When the installation is successful, the message "Installed transport-couchbase into /usr/share/elasticsearch/plugins/transport/couchbase" is logged.
+
+5. Open the Elasticsearch configuration file (**/etc/elasticsearch/elasticsearch.yml**) and add the following to the end of the file. The **username** and **password** should be the one used to connect to Couchbase Server.
```

Comment:
The text about username and password is misleading. What's actually happening is that here in the Elasticseach config the user is *defining* the credentials that must be used when configuring XDCR. When Couchbase makes an XDCR request to the ES plugin, the plugin will check the credentials against the ones defined here in the ES config, and will reject the XDCR request if they don't match.

That said, the next major version of the ES plugin will likely use a different replication scheme, and the username and password *will* need to match the credentials of a Couchbase user with read access to the bucket being replicated.